Server/Test_Plan_OCIS_Compatibility_Test.md

```diff
@@ -0,0 +1,34 @@
+## Compatibility test:
+Note: It is necessary to switch to the latest stable version and do all the steps below.
+After switching to the new ocis version you need to check that all resources, shares and settings still exist and remain unchanged
+
+
+### from version 1 to version 2
```

Comment:
```suggestion
### After upgrade from previous release version to new release version
```

After the discussion in comments above, maybe this can be the heading?